### PR TITLE
Add color example in Readme

### PR DESCRIPTION
**Pull request by oknozor**
addresses #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,34 +42,53 @@ sudo ninja -C build install
...
 ```
 
 Set up your environment so that after updating audio volume, backlight intensity, or whatever, to a new value like 43, it writes that value into the pipe:
 
-```
+```shell script
 echo 43 > /tmp/wobpipe
 ```
 
+Wob will accept values from 0 to `--max` (default 100) and stop running on non-valid input.
```
Comment (PaperMountainStudio) [inline README.md:69]: I am not a native speaker, but wouldn't it be better to write `(by default 100)`? And, `and stop running` could be exit? It sound better to me

File: README.md [hunk 2]
```diff
@@ -42,34 +42,53 @@ sudo ninja -C build install
...
+![wob_colored](doc/img/wob_colored.png)
+
+You can specify colors when passing values to wob : 
+
+```shell script
+echo "25 #FF88c0d0 #FF5e81ac #FFe5e9f0" > /tmp/wobpipe
+```
+
+The first color value represent background color. 
+The second one represent border color.
```
Comment (PaperMountainStudio) [inline README.md:86]: maybe `represents`? and the same on the previous line?

File: README.md [hunk 3]
```diff
@@ -42,34 +42,53 @@ sudo ninja -C build install
...
+```shell script
+echo "25 #FF88c0d0 #FF5e81ac #FFe5e9f0" > /tmp/wobpipe
+```
+
+The first color value represent background color. 
+The second one represent border color. 
+The third one is bar color.
+
+wob colors are composed of a leading `#` followed by height hexadecimal digit. Note that the first two digits 
+represent opacity.
```
Comment (PaperMountainStudio) [inline README.md:90]: `height hexadecimal digit` implies there is only one, maybe mentioning it is in format #AARRGGBB would be enough?

**Comment by oknozor** [comment]
Thank you for the corrections, I am also not a native speaker obviously :)

File: README.md [hunk 4]
```diff
@@ -42,34 +42,52 @@ sudo ninja -C build install
...
+
+```shell script
+echo "25 #FF88c0d0 #FF5e81ac #FFe5e9f0" > /tmp/wobpipe
+```
+
+The first color value represents background color. 
+The second one represents border color.
+The third one is bar color.
+
+wob colors format is `#AARRGGBB` (Note that the first two digits represent opacity)
```
Comment (FlorianFranzen) [inline README.md:89]: Are you sure it is not `#RRGGBBAA`?

Comment (souxd) [reply README.md:89]: The man pages says RRGGBB[AA] @FlorianFranzen

**Pull request closed without merging**